opal/mca/btl/ofi/btl_ofi_component.c

```diff
@@ -314,9 +309,16 @@ static mca_btl_base_module_t **mca_btl_ofi_component_init (int *num_btl_modules,
     rc = fi_getinfo(FI_VERSION(1, 5), NULL, NULL, 0, &hints, &info_list);
     if (0 != rc) {
         BTL_VERBOSE(("fi_getinfo failed with code %d: %s",rc, fi_strerror(-rc)));
+        if (NULL != include_list) {
+            opal_argv_free(include_list);
+        }
         return NULL;
     }
 
+    if (NULL != include_list) {
```

Comment:
Can put this after line 209 if we always want to free include_list after fi_getinfo, then we don't have to duplicate this code.

Reply:
this will be replace with other stuff when PR #7641 comes in.